src/outputgeneration/ParseTreeWriter.js

```diff
@@ -1006,9 +1006,12 @@ export class ParseTreeWriter extends ParseTreeVisitor {
    * @param {NewExpression} tree
    */
   visitNewExpression(tree) {
+    var complexOperand = tree.operand.operand !== void 0;
```

Comment:
I get that, I'm just trying to figure out how to do this in ParseTreeTransformer =)